Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 apache-airflow = "*"
 sagetasks = "*"
 boto3 = "*"
+apache-airflow-providers-amazon = "*"
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** I've started using `~=` for specifying package versions to avoid breaking changes. You don't need to update this PR, but I suggest you adopt this approach the next time you change a dependency. 

What I do is look up the latest version on PyPI. For example, the latest release of the `sevenbridges-python` package is 2.9.1. When I pin the package version, I'll use `sevenbridges-python~=2.9`, which allows for minor and patch updates, but not major updates with potentially backwards-incompatible changes.